extensions/systemd/systemd.js

```diff
@@ -91,15 +91,25 @@ class SystemdProcessManager extends cli.ProcessManager {
             // Systemd prints out "inactive" if service isn't running
             // or "activating" if service hasn't completely started yet
             if (!e.message.match(/inactive|activating/)) {
-                throw e;
+                throw new cli.errors.ProcessError(e);
```

Comment:
Same here as above, thought it might be better to not just throw the original error, but log it properly in the log files.